sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/implementation/AzureCommunicationChatServiceImplBuilder.java

```diff
@@ -72,6 +87,17 @@ public AzureCommunicationChatServiceImplBuilder pipeline(HttpPipeline pipeline)
      */
     private JacksonSerder jacksonSerder;
 
+    /**
+     * Sets The serializer to serialize an object into a string.
+     *
+     * @param jacksonSerder the jacksonSerder value.
+     * @return the AzureCommunicationChatServiceImplBuilder.
+     */
+    public AzureCommunicationChatServiceImplBuilder jacksonSerder(JacksonSerder jacksonSerder) {
+        this.jacksonSerder = jacksonSerder;
+        return this;
```

Comment:
We shouldn't take `JacksonSerder` from the user, we'll use the one created internally. It's not a contract  (interface) anymore but a final class.

Reply:
This will require code generator change. Let's do this manually for now to move forward before code generator can catch up.

Reply:
Update code generator to address this and pushed update